### PR TITLE
draft of German translation

### PR DESCRIPTION
**Pull request by sushi2k**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [ ] Your PR is linked to an issue.
- [ ] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [ ] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [x] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #320.

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit d970bbaba0cc72344ccfb1634e8497513555f676: Broken link result: 0 .
  
 Markdown result errorlength: 2 .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 68160799c6a587e3422a49a347a6ed9e42b06f40: Broken link result: 0 .
  
 Markdown result errorlength: 2 .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 18c804b5de61f36b6fcfc9931344e421ea54ecb5: Broken link result: 0 .
  
 Markdown result errorlength: 2 .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document-de/0x06-V1-Architecture_design_and_threat_modelling_requireme.md [hunk 1]
```diff
@@ -35,7 +37,7 @@ Für weitere Informationen:
 
 - OWASP Mobile Top 10: M10 (Extraneous Functionality) - <https://www.owasp.org/index.php/Mobile_Top_10_2016-M10-Extraneous_Functionality>
 - OWASP Security Architecture cheat sheet - <https://www.owasp.org/index.php/Application_Security_Architecture_Cheat_Sheet>
-- OWASP Thread modelling - <https://www.owasp.org/index.php/Application_Threat_Modeling>
+- OWASP Thread Modeling - <https://www.owasp.org/index.php/Application_Threat_Modeling>
 - OWASP Secure SDLC Cheat Sheet - <https://www.owasp.org/index.php/Secure_SDLC_Cheat_Sheet>
 - Microsoft SDL - <https://www.microsoft.com/en-us/sdl/>
 - NIST SP 800-57 (Recommendation for Key Management) - <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
```
Comment (commjoen) [inline Document-de/0x06-V1-Architecture_design_and_threat_modelling_requireme.md]: Misisng...
```suggestion
- NIST SP 800-57 (Recommendation for Key Management) - <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
- security.txt - <https://securitytxt.org/>
```

File: Document-de/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,25 @@
...
+- Anforderung MSTG-AUTH-11 wurde aktualisiert um "kontextbezogene Informationen".
+- Anforderung MSTG‑PLATFORM‑10 wurde hinzugefügt für einen weiteren Fall um WebViews sicher zu verwenden.
+- Anforderung MSTG‑AUTH‑12 wurde hinzugefügt um Entwickler daran zu erinnern Authorisierung zu implementieren, vor allem im Fall von multi-user Apps.
+- Beschreibung hinzugefügt, wie der MASVS in einem Risk Assessment verwendet werden sollte.
+- Beschreibung hinzugefügt für Bezahl-Inahlte.
+- Anforderung MSTG‑ARCH‑11 wurde hinzugefügt um Responsible Disclosure Policies zu implementieren.
+- Anforderung MSTG‑ARCH‑12 wurde hinzugefügt um Datenschutzgesetze zu befolgen.
+- Anforderung MSTG‑PLATFORM‑11 wurde hinzugefügt um 3rd Party Tastaturen zu verbieten.
+- Anforderung MSTG‑RESILIENCE‑13 wurde hinzugefügt um das Abhören von Apps zu erschweren.
+
```
Comment (commjoen) [inline Document-de/CHANGELOG.md]: Can you please add the missing sentence on the translations (translated to german ;-)) ? `- Updated the MASVS in the following languages: Chinese (ZHTW), English, German, French, Russian, Spanish, and Japanese).`

**Comment by OMTGreleaser** [comment]
Results for commit 9b3815c036023141ebf5e31044f2b8109b368db3: Broken link result: 0 .
  
 Markdown result errorlength: 2 .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document-de/CHANGELOG.md (hunk 2 above)
Comment (sushi2k) [reply Document-de/CHANGELOG.md]: done

**Comment by sushi2k** [comment]
Thanks Jeroen!

**Comment by OMTGreleaser** [comment]
Results for commit fda0abe0f55b70260f96bf92b625ab90c36f1a68: Broken link result: 0 .
  
 Markdown result errorlength: 1 .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document-de/0x06-V1-Architecture_design_and_threat_modelling_requireme.md [hunk 3]
```diff
@@ -25,6 +25,8 @@ Die Anforderungen für MASVS-L1 und MASVS-L2 sind nachfolgend aufgelistet.
 | **1.8** | MSTG‑ARCH‑8 | Eine dedizierte Richtlinie zum Management von kryptographischen Schlüsseln (falls in der App genutzt) beschreibt den sicheren Umgang mit Schlüsseln über den gesamten Lebenszyklus, idealerweise basierend auf Standards wie NIST SP 800-57. |   | ✓ |
 | **1.9** | MSTG‑ARCH‑9 | Es gibt einen Mechanismus in der mobilen App um App-Aktualisierungen zu erzwingen. |   | ✓ |
 | **1.10** | MSTG‑ARCH‑10 | Security wird in allen Teilen des Softwareentwicklungszyklus berücksichtigt. |   | ✓ |
+| **1.11** | MSTG‑ARCH‑11 | Eine "Responsible Disclosure Policy" ist vorhanden und wird umgesetzt. |   | ✓ |
+| **1.12** | MSTG‑ARCH‑12 | Die App sollte Datenschutzgetze und Regulierungen befolgen. | ✓ | ✓ |
```
Comment (codethatrocks) [inline Document-de/0x06-V1-Architecture_design_and_threat_modelling_requireme.md]: Typo: Datenschutzgetze  -> Datenschutzgesetze

File: Document-de/0x06-V1-Architecture_design_and_threat_modelling_requireme.md [hunk 4]
```diff
@@ -35,7 +37,8 @@ Für weitere Informationen:
 
 - OWASP Mobile Top 10: M10 (Extraneous Functionality) - <https://www.owasp.org/index.php/Mobile_Top_10_2016-M10-Extraneous_Functionality>
 - OWASP Security Architecture cheat sheet - <https://www.owasp.org/index.php/Application_Security_Architecture_Cheat_Sheet>
-- OWASP Thread modelling - <https://www.owasp.org/index.php/Application_Threat_Modeling>
+- OWASP Thread Modeling - <https://www.owasp.org/index.php/Application_Threat_Modeling>
```
Comment (codethatrocks) [inline Document-de/0x06-V1-Architecture_design_and_threat_modelling_requireme.md]: Typo: Thread Modeling -> Threat Modeling

File: Document-de/0x07-V2-Data_Storage_and_Privacy_requirements.md [hunk 5]
```diff
@@ -34,6 +34,9 @@ Ein Großteil von Datenpannen kann bereits durch Einhaltung einfacher Regeln ver
 | **2.10** | MSTG‑STORAGE‑10 | Die App hält sensible Daten nur solange wie nötig im Speicher und betroffene Speicherbereiche werden nach Nutzung explizit gelöscht. |  | ✓ |
 | **2.11** | MSTG‑STORAGE‑11 | Die App erzwingt ein Minimum an Geräteschutz-Richtlinien wie das Definieren eines Gerätepassworts. |  | ✓ |
 | **2.12** | MSTG‑STORAGE‑12 | Die App klärt den Nutzer über die Art und Weise der verarbeiteten personenbezogenen Daten auf und gibt dem Nutzer Security-Best-Practice-Empfehlungen zum Umgang mit der App. |  | ✓ |
+| **2.13** | MSTG‑STORAGE‑13 | Sensible Daten sollten nicht lokal auf dem mobile Gerät gespeichert werden. Daten sollten stattdessen direkt vom API-Endpunkt abgerufen und nur im Arbeitsspeicher vorgehalten werden. |  | ✓ |
```
Comment (codethatrocks) [inline Document-de/0x07-V2-Data_Storage_and_Privacy_requirements.md]: auf dem mobile Gerät -> auf dem mobilen Gerät

File: Document-de/0x07-V2-Data_Storage_and_Privacy_requirements.md [hunk 6]
```diff
@@ -34,6 +34,9 @@ Ein Großteil von Datenpannen kann bereits durch Einhaltung einfacher Regeln ver
...
+| **2.14** | MSTG‑STORAGE‑14 | Falls doch sensible Daten lokal vorgehalten werden müssen, sollten diese mit einem Schlüssel verschlüsselt werden der im vom jeweiligen Betriebssystem angebotenen sicheren Speichermechanismus abgelegt ist. Der Zugriff auf den Schlüssel sollte authentifizierung erfordern. |  | ✓ |
```
Comment (codethatrocks) [inline Document-de/0x07-V2-Data_Storage_and_Privacy_requirements.md]: Authentifizierung mit groß A ;)

File: Document-de/0x07-V2-Data_Storage_and_Privacy_requirements.md [hunk 7]
```diff
@@ -34,6 +34,9 @@ Ein Großteil von Datenpannen kann bereits durch Einhaltung einfacher Regeln ver
...
+| **2.15** | MSTG‑STORAGE‑15 | Der lokale Speicher von der App sollte gelöscht werden, nach erreichen einer exzessiven Anzahl von fehlgeschlagenen Login-Versuchen.  |  | ✓ |
```
Comment (codethatrocks) [inline Document-de/0x07-V2-Data_Storage_and_Privacy_requirements.md]: nach Erreichen mit groß E

File: Document-de/0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md [hunk 8]
```diff
@@ -48,19 +48,23 @@ Folgende Eckpunkte gelten:
 | **8.11** | MSTG‑RESILIENCE‑11 | Alle ausführbaren Dateien und Bibliotheken der App sind entweder auf Dateiebene verschlüsselt und/oder wichtige Code- und Datenabsegmente in ausführbaren Dateien sind verschlüsselt oder durch Packing obfuskiert. Triviale statische Analyse offenbart keinen wichtigen Code oder Daten. | ✓ |
 | **8.12** | MSTG‑RESILIENCE‑12 | Wenn das Ziel der Obfuskierung der Schutz sensibler Logik wie Algorithmen oder Berechnungen ist, so wird ein angemessener Obfuskierungsmechanismus, dem Stand der Technik entsprechend, genutzt der resilient gegen manuelle und automatisierte De-Obfuskierungsangriffe ist. Die Wirksamkeit der Obfuskierungsmethode muss durch manuelle Tests überprüft werden. Es ist zu beachten, dass hardware-basierte Isolations-Mechanismen softwarebasierter Obfuskierung vorzuziehen sind. | ✓ |
 
+### Abhören erschweren
+
+| **8.13** | MSTG‑RESILIENCE‑13 | Neben einer gehärteten Kommunikation zwischen client und API-Endpunkt, sollte auch die übertragene Payload verschlüsselt werden um das Abhören von Daten zu erschweren. | ✓ |
```
Comment (codethatrocks) [inline Document-de/0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md]: + | **8.13** | MSTG‑RESILIENCE‑13 | Neben einer gehärteten Kommunikation zwischen Client und API-Endpunkt, sollte auch der übertragene Payload verschlüsselt werden um das Abhören von Daten zu erschweren. | ✓ |

File: Document-de/CHANGELOG.md [hunk 9]
```diff
@@ -2,6 +2,25 @@
 
 Dieses Dokument wurde automatisch erstellt am {{gitbook.time}}.
 
+## V1.2-RC - Pre-Release
+
+Die folgenden Änderungen sind Teil von Version 1.2-RC:
+
+- "Flagship Status" von OWASP erhalten.
+- Anforderung MSTG-Storage-1 verändert.
+- Anforderung MSTG-STORAGE-13, MSTG-STORAGE-14 und MSTG-STORAGE-15 wurden hinzugefügt mit einem Fokus auf Datenschutz.
```
Comment (codethatrocks) [inline Document-de/CHANGELOG.md]: - Anforderung MSTG-STORAGE-13, MSTG-STORAGE-14 und MSTG-STORAGE-15 wurden hinzugefügt mit Fokus auf Datenschutz.

File: Document-de/CHANGELOG.md [hunk 10]
```diff
@@ -2,6 +2,25 @@
...
+- Anforderung MSTG‑CODE‑4 wurde aktualisiert um mehr Fälle als nur Debugging abzudecken.
+- Anforderung MSTG-AUTH-11 wurde aktualisiert um "kontextbezogene Informationen".
+- Anforderung MSTG‑PLATFORM‑10 wurde hinzugefügt für einen weiteren Fall um WebViews sicher zu verwenden.
+- Anforderung MSTG‑AUTH‑12 wurde hinzugefügt um Entwickler daran zu erinnern Authorisierung zu implementieren, vor allem im Fall von multi-user Apps.
```
Comment (codethatrocks) [inline Document-de/CHANGELOG.md]: Autorisierung

File: Document-de/CHANGELOG.md [hunk 11]
```diff
@@ -2,6 +2,25 @@
...
+- Beschreibung hinzugefügt, wie der MASVS in einem Risk Assessment verwendet werden sollte.
+- Beschreibung hinzugefügt für Bezahl-Inahlte.
```
Comment (codethatrocks) [inline Document-de/CHANGELOG.md]: Typo: Bezahl-Inhalte

**Comment by sushi2k** [comment]
Thanks @codethatrocks ! 

Just fixed all your feedback and pushed again. Please approve :-)

**Comment by OMTGreleaser** [comment]
Results for commit a7de788d2183bfd3f1e17458713b433e6a920138: Broken link result: 0 .
  
 Markdown result errorlength: 1 .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**